README.md

```diff
@@ -126,11 +126,26 @@ Before installation, please make sure you have already installed the following t
    > Check this video of how to do that: https://www.youtube.com/watch?v=D0U8vD8m1I0  
    > Note: If using Docker, the MongoDB port is **27018**
 
-    In the client ```.env ```  to use the image processing functionality obtain a service account key with Cloud Vision API access
-      > Follow the steps in the [Google Cloud documentation](https://cloud.google.com/iam/docs/keys-create-deletet) to create a service account and download the JSON key file.  
-      Place the file key ``` GOOGLE_APPLICATION_CREDENTIALS = '' ```
+7.  To use the image processing functionality obtain a service account key with Cloud Vision API access Follow the steps: (You should have an account on google cloud also enable billing ``` its free for 90 days ``` ).
```

Comment:
this is a problem, if it's free for 90 days, unless you are going to sponsor it, we need to use a free service or library

Reply:
I wrote for the contributor,  to get that key, because if they don't do register for billing, key don't work.
Should i delete that?

Reply:
cuz even if it's a contributor or not, it's still a 90 days limit, try a library that's free

Reply:
btw nsfwjs is for frontend, is that where you tried it?, btw let's move this conversation to the discord server, tag me with the answer on discord when you see it

Reply:
I tried to use ```nsfwjs``` but there is an error while installing the ```npm install @tensorflow/tfjs-node```, I couldn't resolve, plz if can you figure out how to install...
There is an api of [ moderatecontent ](https://moderatecontent.com/), Its working fine. can you once check their free service...